monkestation/code/modules/martial_arts/tribal_claw.dm

```diff
@@ -0,0 +1,119 @@
+//ported directly from Bee, cleaned up and updated to function with TG. thanks bee!
+
+#define TAIL_SWEEP_COMBO "DH"
+#define FACE_SCRATCH_COMBO "HH"
+#define JUGULAR_CUT_COMBO "HD"
+#define TAIL_GRAB_COMBO "DDG"
+
+/datum/martial_art/tribal_claw
+	name = "Tribal Claw"
+	id = MARTIALART_TRIBALCLAW
+	allow_temp_override = FALSE
+	help_verb = /mob/living/carbon/human/proc/tribal_claw_help
+
+/datum/martial_art/tribal_claw/proc/check_streak(mob/living/carbon/human/attacker, mob/living/carbon/human/target)
+	if(findtext(streak,TAIL_SWEEP_COMBO))
+		streak = ""
+		tailSweep(attacker,target)
+		return TRUE
+	if(findtext(streak,FACE_SCRATCH_COMBO))
+		streak = ""
+		faceScratch(attacker,target)
+		return TRUE
+	if(findtext(streak,JUGULAR_CUT_COMBO))
+		streak = ""
+		jugularCut(attacker,target)
+		return TRUE
+	if(findtext(streak,TAIL_GRAB_COMBO))
+		streak = ""
+		tailGrab(attacker,target)
+		return TRUE
+	return FALSE
+
+//Tail Sweep, triggers an effect similar to Alien Queen's tail sweep but only affects stuff 1 tile next to you, basically 3x3.
+/datum/martial_art/tribal_claw/proc/tailSweep(mob/living/carbon/human/attacker, mob/living/carbon/human/target)
+	if(attacker == current_target)
+		return
+	log_combat(attacker, target, "tail sweeped(Tribal Claw)", name)
+	target.visible_message("<span class='warning'>[attacker] sweeps [target]'s legs with their tail!</span>", \
+						"<span class='userdanger'>[attacker] sweeps your legs with their tail!</span>")
+	var/static/datum/action/cooldown/spell/aoe/repulse/xeno/tail_sweep = new
```

Comment:
make a variant of the spell without a cooldown to put here to be safe

Reply:
copied xeno tail sweep into a new file so it can be modified to whatever people wish
cooldown removed